test/kubemark/start-kubemark-master.sh

```diff
@@ -15,18 +15,25 @@
 # limitations under the License.
 
 # TODO: figure out how to get etcd tag from some real configuration and put it here.
+
+EVENT_STORE_IP=$(cat /tmp/event-store-ip)
```

Comment:
Well, I can put it in the .bashrc (or something), but I'm not sure it is any better. And I'd rather avoid using salt.